pom.xml

```diff
@@ -480,18 +437,6 @@
     </reporting>
 
     <profiles>
-        <profile>
```

Comment:
Reinstate this please.

Reply:
Or add the modules we do want in the release profile instead

Reply:
Fixed in 885359c